mergify_engine/tests/functional/actions/test_squash.py

```diff
@@ -34,23 +34,18 @@ async def test_squash_several_commits_ok(self):
 
         await self.setup_repo(yaml.dump(rules))
 
-        repo_name = "fork"
-        branch_name = self.get_full_branch_name(f"{repo_name}/pr_squash_test")
+        branch_name = self.get_full_branch_name("pr_squash_test")
 
         await self.git(
-            "checkout",
-            "--quiet",
-            f"{repo_name}/{self.main_branch_name}",
-            "-b",
-            branch_name,
+            "checkout", "--quiet", f"origin/{self.main_branch_name}", "-b", branch_name
         )
 
         for i in range(0, 3):
             open(self.git.tmp + f"/file{i}", "wb").close()
             await self.git("add", f"file{i}")
             await self.git("commit", "--no-edit", "-m", f"feat(): add file{i+1}")
 
-        await self.git("push", "--quiet", repo_name, branch_name)
+        await self.git("push", "--quiet", "fork", branch_name)
```

Comment:
This seems unrelated at first sight?

Reply:
some tests what using the main branch of the fork instead of the main branch of the origin repository.
Since I don't create the main anymore on the fork, I have to change all tests using the main branch of the fork.